### ISSUE TITLE
Website formatting

### ISSUE THREAD

**Issue by ChrisRackauckas** [SciML/sciml.ai#80]
@logankilpatrick the new homepage is fantastic! Though there seem to be some formatting issues:

- Logo's a little small on the left
- The navbar has a ton on one line and few on the other
- There's lots of vertical spacing between the elements below
- Centering the sponsors might look better
- [About (coming soon)](https://sciml.ai/#!) Can probably be deleted.

![Screenshot 2022-03-22 163933](https://user-images.githubusercontent.com/1814174/159571821-c0c73d9f-21e0-4760-a3fa-459a948244f1.png)

**Comment by logankilpatrick** [comment]
Thanks! Still working through the kinks, I will try to get these fixed, it is kinda tough on some of them since I am fighting to keep the original franklin CSS along with the new CSS and making sure they play well so that the other pages don't get ruined

**Comment by logankilpatrick** [comment]
The logo size can't really change much in this context because it is part of the background image and the text moves dynamically. The padding between the rows is also something I could not figure out. The rest have been addressed.

**Comment by ViralBShah** [comment]
This design is in the right direction, but needs some refinement. I think it should be reasonably quick to fix.

The links on top need to be better formatted. Or perhaps we need to use menus that drop down or something. It looks a bit weird, and worse in certain browser window sizes.

There's also a huge amount of whitespace as you scroll down. The icons are too small, and the text is in a narrow band and voluminous - which makes it mostly look empty as you scroll.

**Comment by Mxyank24** [comment]
Everything is looking good but we can move that ( github stars ) to the top right of the banner and also add day / night toggle option

<img width="1470" height="956" alt="Image" src="https://github.com/user-attachments/assets/25e70a19-41eb-4ad2-a23b-3bb38c4bc588" />

**Comment by Mxyank24** [comment]
> Everything is looking good but we can move that ( github stars ) to the top right of the banner and also add day / night toggle option
> 
> <img alt="Image" width="1470" height="956" src="https://private-user-images.githubusercontent.com/282147947/598795084-25e70a19-41eb-4ad2-a23b-3bb38c4bc588.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Nzk4ODU4MTcsIm5iZiI6MTc3OTg4NTUxNywicGF0aCI6Ii8yODIxNDc5NDcvNTk4Nzk1MDg0LTI1ZTcwYTE5LTQxZWItNGFkMi1hMjNiLTNiYjM4YzRiYzU4OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwNTI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDUyN1QxMjM4MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05OGQxODcwODVlNjZiMTZiNGQ1YWQ5ZGYwMzFlOWU1MzdkYjhkMzMxNTFiNTk0ZjlmNzQ1YzNmY2Y3MzM3ZGI0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZyZXNwb25zZS1jb250ZW50LXR5cGU9aW1hZ2UlMkZwbmcifQ.-g6QwbdGdYIEQyG8owVxaY2nbE-A5Uzavh7X-Y1WkW4">

What is your opinion? @ChrisRackauckas

**Comment by ChrisRackauckas** [comment]
I don't think it needs light/dark mode, it's fine. the github stars is a little small but I don't think it's a major deal. Moving it and reducing some whitespace can be fine

**Comment by Mxyank24** [comment]
oh okie

**Comment by Mxyank24** [comment]
is something odd with github star bt ?

<img width="1470" height="878" alt="Image" src="https://github.com/user-attachments/assets/521655d1-4480-4883-9ebf-7184f8606f0f" />

**Comment by ChrisRackauckas** [comment]
Requests are timing out, yeah.
